packages/ndla-icons/src/action/AddLine.tsx

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
```

Comment:
Kanskje oppdatere scriptet slik at årstallet blir korrekt.

Reply:
Eventuelt lese datostempel fra svg-fila og bruke det i generert tsx.

Reply:
Endring av COPYRIGHT-fila gjør at samtlige får nye årstall (fordi det kjøres for alt) så har lagt inn en regex som replacer årstallet der hvor det trengs. Har også oppdatert ' til " for index-filene slik at ikke de får stor diff hver gang scriptet kjøres.

Det som dog skjer, er at sida jeg gikk litt langt til å begynne med (og byttet ut ikoner før jeg måtte legge dem inn igjen) så er det noen av ikonene vi har i dag som dermed har nytt created-timestamp og derfor også får oppdatert årstall til 2024. Det har neppe noe å si long term, sida de uansett skal fjernes igjen iløpet av høsten til fordel for nye ikoner.